cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -71,13 +71,20 @@
             <ul class="list list__links">
             {% for link in value.links %}
                 <li class="list_item">
+                    {% if link.text is defined and link.text.find('@') > -1 %}
```

Comment:
Yeah, we can use Djangos [validate_email](https://github.com/django/django/blob/12b4280444b58c94197255655e284e4103fe00a9/django/core/validators.py#L239) method but I'm sure where that code would need to reside. Would it be on the the Hyperlink molecule? A context function?